### DISCUSSION TITLE
How do I handle phone numbers?

### DISCUSSION THREAD

**Discussion by davidgtu** [vercel/next.js#13580] [Help] [answered]
I have a Link component that uses Next/Link and I'm passing in a telephone number formatted like this: `tel:+15555555555`

However I get an error:

```js
Invalid href passed to router: tel:+15555555555
```

Are telephone numbers not supported?

**Comment by balazsorban44** [comment, marked as answer]
From the [docs](https://nextjs.org/docs/api-reference/next/link) :

> External URLs, and any links that don't require a route navigation using /pages, don't need to be handled with Link; use the anchor tag for such cases instead.

With other words, you do not need a next link for tel. just use a regular `a` with `href`

(if you found my answer helpful, could you please mark this as ansewered 🙂)
